ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -650,5 +668,8 @@ public RouterConfig(VerifiableProperties verifiableProperties) {
         verifiableProperties.getBoolean(ROUTER_PUT_REQUEST_USE_JAVA_NATIVE_CRC32, false);
     operationController =
         verifiableProperties.getString(OPERATION_CONTROLLER, "com.github.ambry.router.OperationController");
+    routerRequestHandlerNumOfThreads = verifiableProperties.getInt(ROUTER_REQUEST_HANDLER_NUM_OF_THREADS, 7);
+    routerStoreKeyConverterFactory = verifiableProperties.getString(ROUTER_REQUEST_HANDLER_NUM_OF_THREADS,
+        "com.github.ambry.store.StoreKeyConverterFactoryImpl");
   }
```

Comment:
Can we add one unit test to verify, the thread number and StoreKeyConverter are used as specified by the verifiableProperties.ROUTER_REQUEST_HANDLER_NUM_OF_THREADS and verifiableProperties.ROUTER_STORE_KEY_CONVERTER_FACTORY.

Not sure if it's easy. If it's not, manual test is ok too.

Reply:
I add a test to my local and test it. but I feel it's not necessary to add to the central test list.